Пофиксил проверку URI

dvakor opened the pull request:
Фиксит #957 

Вип по причине отсутствия тестов

dvakor committed:
Пофиксил проверку URI

EvilBeaver commented:
@ArchLord42RU заливать?

dvakor commented:
> @ArchLord42RU заливать?

Без тестов меня совесть будет мучать, давая я точно на этих выходных тесты запилю и кину :)
Работа душит чето :(

EvilBeaver commented:
Ну оно как бы критикал баг, так что ты там это... жену на дачу, детей в пионерский лагерь и за комп :)

dvakor committed:
Тесты

Добавлены тесты проверки копирования \ перемещения файлов с относительным путем

dvakor commented:
> Ну оно как бы критикал баг, так что ты там это... жену на дачу, детей в пионерский лагерь и за комп :)

Ну вышло по другому, погода сказала на тебе -27, сиди пили :) В общем сделяль
